app/config/routing.yml

```diff
@@ -5,8 +5,8 @@
 # the routes in YAML, XML or PHP files.
 # See http://symfony.com/doc/current/book/routing.html
 app:
-    resource: '@AppBundle/Controller/'
-    type:     annotation
+    resource: '@AppBundle/Action/'
```

Comment:
IMO, keeping the controllers in ``Controller`` is better, even if you rely on a special loader aware of the way they are registered as services.